Examples/Scripts/FreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash 
+
+get_usage_and_exit(){
+    echo "usage: "
+    echo "FreeSurferPipelineBatch-long.sh [options]"
+    echo "options:"
+    echo "  --runlocal                      run locally [FALSE]"
+    exit -1
+}
+
+command_line_specified_run_local=FALSE
+while [ -n "$1" ]; do
+    case "$1" in
+        --runlocal) shift; command_line_specified_run_local=TRUE ;;
+        *) shift ;;
+    esac
+done
+
+#################################################################################################
+# General input variables
+##################################################################################################
+#Location of Subject folders (named by subjectID)
+StudyFolder="/media/myelin/brainmappers/Connectome_Project/MishaLongitudinal/hcp/FSLong-test" 
```

Comment:
We probably don't want to commit any myelin paths into the pipelines examples.

Reply:
Agree.